lms/envs/common.py

```diff
@@ -1247,7 +1247,8 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
```

Comment:
@cahrens Did you have thoughts on this? @andy-armstrong Do you know if the modernizr package that's in the platform already includes "fontface" and "generatedcontent"? If so, we can omit this one entirely.